index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Diwakar Shrestha Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="nav">
+        <ul>
+          <li class="profile-name">Diwakar Shrestha</li>
+          <!-- <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+          <li class="mail">
+            <img src="images/mail.png" alt="contact" />
+          </li> -->
+          <li><img src="images/toggle.png" alt="toggle-bar" /></li>
+        </ul>
+      </header>
+
+      <section class="headline">
+        <p class="name"><strong>Diwakar Shrestha</strong></p>
```

Comment:
- [ ] kindly change the `p` tag to `h1` tag as requested in the project requirement. `h1` as the main title is good for semantic and SEO reasons. kindly check this [HTML and CSS best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)